queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Factories/SignatureItemFactoryDEFAULT.cs

```diff
@@ -16,14 +17,15 @@ public class SignatureItemFactoryDEFAULT : SignatureItemFactory
             CurrencyDecimalDigits = 2
         };
         
-        public List<SignaturItem> GetSignaturesForPosReceiptTransaction(string cashBoxId, string receiptId, decimal sumOfPayItems, string previousHash, long ftReceiptCase)
+        public List<SignaturItem> GetSignaturesForPosReceiptTransaction(Guid cashBoxId, Guid receiptId,
+            decimal sumOfPayItems, string receiptCase, long ftReceiptCase)
```

Comment:
```suggestion
            decimal sumOfPayItems, string previousHash, long ftReceiptCase)
```
🤔 I think this is not the receipt case.. below you pass the hash